dev-haskell/hackage-security/hackage-security-0.6.0.1-r1.ebuild.orig

```diff
@@ -0,0 +1,79 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# ebuild generated by hackport 0.6.7.9999
+#hackport: flags: -mtl21,+use-network-uri
+
+CABAL_FEATURES="lib profile haddock hoogle hscolour test-suite"
+inherit haskell-cabal
+
+DESCRIPTION="Hackage security library"
+HOMEPAGE="https://github.com/haskell/hackage-security"
+SRC_URI="https://hackage.haskell.org/package/${P}/${P}.tar.gz
+	https://hackage.haskell.org/package/${P}/revision/4.cabal -> ${PF}.cabal"
+
+LICENSE="BSD"
+SLOT="0/${PV}"
+# keep in sync with ghc-8.10
+KEYWORDS="~amd64 ~x86"
+<<<<<<< HEAD:dev-haskell/hackage-security/hackage-security-0.6.0.1.ebuild
+IUSE=""
+
+RESTRICT=test
+=======
```

Comment:
Looks like a conflict in a stray file.

Reply:
Gahh so close. How did repoman miss this?

Reply:
fixed